src/azul/dss.py

```diff
@@ -220,6 +223,16 @@ def __init__(self, *args, num_workers: int = None, **kwargs):
                          **kwargs)
 
 
+# Cached to prevent exceeding requests limits during TDR reindex
```

Comment:
That's a bit of in implementation detail of the call site. The reason for caching is universal: a resources is expensive to create and frequently used.